tests/phpunit/elementor/modules/site-navigation/data/recent/test-controller.php

```diff
@@ -57,10 +57,14 @@ public function test_get_items__invalid_args( $posts_per_page, $post_type, $post
 	public function get_items_provider() {
 		return [
 			'no params' => [
-				'', '', '',
+				'',
+				'',
+				'',
```

Comment:
```suggestion
				'posts_per_page' => '',
				'post_type' => '',
				'post__not_in' => '',
```

I think it will be clearer

Reply:
Not mine, but i can change it, NP